Pipfile

```diff
@@ -4,25 +4,25 @@ url = "https://pypi.org/simple"
 verify_ssl = true
 
 [dev-packages]
-pre-commit = "~=2.6"
+pre-commit = "~=2.10"
```

Comment:
could dependabot do that?

Reply:
Yes.
I don't remember the reason to stop using it now, but previously we used it.

Can we use it again? > @KazuCocoa 

https://github.com/appium/python-client/pulls?q=is%3Apr+is%3Aclosed+author%3Aapp%2Fdependabot-preview